lib/gui/app.js

```diff
@@ -298,11 +298,18 @@ app.controller('HeaderController', function(SelectionStateModel, OSOpenExternalS
 
 });
 
-app.controller('StateController', function($state) {
+app.controller('StateController', function($rootScope, $state) {
 
   /**
    * @param {string} state - state page
```

Comment:
Looks like I failed to catch it last time, but this should also include all the JSDoc tags mentioned below.